schemas/context/identitymap.schema.json

```diff
@@ -0,0 +1,42 @@
+{
+  "meta:license": [
+    "Copyright 2018 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/context/identitymap",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Profile",
+  "type": "object",
+  "auditable": true,
+  "meta:extensible": true,
+  "meta:intendedToExtend": ["https://ns.adobe.com/xdm/context/profile", "https://ns.adobe.com/xdm/context/pexperienceevent"],
+  "description":
+    "End User identity assertions.",
+  "definitions": {
+    "identitymap": {
+      "properties": {
```

Comment:
@cdegroot-adobe / @kstreeter , I thought this was supposed to use the new map type support.